kvmagent/kvmagent/plugins/port_mirror_plugin.py

```
@@ -99,53 +101,76 @@ def _delete_gre_device(self, tunnel, rec_device_name):
 
-    def _set_redirect_config(self, device_name, mirror_device_name, source_vlan_id=0):
-        self._set_mirror_src_config(device_name, mirror_device_name, "Egress", source_vlan_id)
+    def _set_redirect_config(self, device_name, mirror_device_name, interface_name, source_vlan_id=0):
+        self._set_mirror_src_config(device_name, mirror_device_name, "Egress", interface_name, source_vlan_id)
     def _clear_redirect_config(self, device_name, mirror_device_name):
         self._clear_mirror_src_config(device_name, mirror_device_name, "Egress")
 
-    def _set_mirror_src_config(self, device_name, mirror_device_name, direction, source_vlan_id=0):
+    def _set_mirror_src_config(self, device_name, mirror_device_name, direction, interface_name, source_vlan_id=0):
+        if not interface_name:
+            raise ValueError("interface_name must be specified to set mirror source config")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)